src/js/structs/ServiceTree.js

```diff
@@ -168,21 +169,40 @@ module.exports = class ServiceTree extends Tree {
   }
 
   getServiceStatus() {
-    let {tasksRunning} = this.getTasksSummary();
-    let deployments = this.getDeployments();
+    // Return the service status with the highest index
+    let treeStatus = this.getStatusSummary().reduce((memo, count, statusKey) => {
+      if (count > 0) {
+        Object.keys(ServiceStatus).forEach(statusId => {
```

Comment:
Do we really need to iterate through the `ServiceStatus` object over and over again? How about creating a temporary map, or adding a different getter?